unity-renderer/Assets/Scripts/MainScripts/DCL/DataStore/DataStore.cs

```diff
@@ -29,7 +29,7 @@ public class DataStore
         public readonly BaseVariable<NFTPromptModel> onOpenNFTPrompt = new BaseVariable<NFTPromptModel>();
         public readonly DataStore_AvatarsLOD avatarsLOD = new DataStore_AvatarsLOD();
 
-        public class BuilderInWorld
+        public class DataStore_BuilderInWorld
```

Comment:
```suggestion
        public class DataStore_BuilderInWorld
```
This class shouldn't be on its own file?

Reply:
Currently, I am following the same convention that we have for Quest, HUDs, Player and AvatarLOD, all those classes are in the same file